tests/test-config.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright 2023-2024 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
We can expunge "All Rights Reserved" from both our code and our memories. 😄 It's not been needed since at least the year 2000...if not 1952... https://en.wikipedia.org/wiki/All_rights_reserved

Reply:
Dropped all rights reserved here: https://github.com/w3c/vc-di-bbs-test-suite/pull/13/commits/757d9fef02ceba33342ce60de8eede143beb1588

WRT to your comments above about the sails vectors, the vc 2.0 one is used in the `verify` suite. It goes by the name `subjectHasArrays` in that suite. The verifier test data is similarly overly complicated and as confusing as the ecdsa sd verifier test data is sadly.

Reply:
Versioned VCs are used in this PR: https://github.com/w3c/vc-di-bbs-test-suite/pull/16 and in https://github.com/w3c/vc-di-bbs-test-suite/pull/17 both of which target this branch.